### PR TITLE
TELCODOCS-303: Added additional resources at the bottom of the assembly with info about the operator in disconnected mode

### PR DESCRIPTION
**Pull request by MirzWeiss**
Documentation for [KATA-752](https://issues.redhat.com/browse/KATA-752).

Documentation preview: https://deploy-preview-35960--osdocs.netlify.app/openshift-enterprise/latest/sandboxed_containers/deploying-sandboxed-container-workloads?utm_source=github&utm_campaign=bot_dp#additional-resources

@jensfr @zanetworker @dmaizel

Approved by SMEs and QE.

### CODE REVIEW

**Comment by MirzWeiss** [comment]
Documentation for [KATA-752](https://issues.redhat.com/browse/KATA-752).

Documentation preview: https://deploy-preview-35960--osdocs.netlify.app/openshift-enterprise/latest/sandboxed_containers/deploying-sandboxed-container-workloads?utm_source=github&utm_campaign=bot_dp

@jensfr @zanetworker @dmaizel

File: sandboxed_containers/deploying-sandboxed-container-workloads.adoc [hunk 1]
```diff
@@ -37,3 +37,8 @@ include::modules/sandboxed-containers-triggering-installation-kata-runtime.adoc[
 include::modules/sandboxed-containers-selecting-nodes.adoc[leveloffset=+3]
 include::modules/sandboxed-containers-scheduling-workloads.adoc[leveloffset=+3]
 include::modules/sandboxed-containers-viewing-workloads-from-cli.adoc[leveloffset=+2]
+
+== Additional resources
+
+* The {sandboxed-containers-operator} is supported in a restricted network environment. For more information, xref:../operators/admin/olm-restricted-networks.adoc#olm-restricted-networks[Using Operator Lifecycle Manager on restricted networks].
+* When using a disconnected cluster on a restricted network, you must xref:../operators/admin/olm-configuring-proxy-support.adoc#olm-configuring-proxy-support[configure proxy support in Operator Lifecycle Manager] to access the Red Hat-provided OperatorHub. Using a proxy allows the cluster to fetch the {sandboxed-containers-operator}.
```
Comment (sagidlow) [inline sandboxed_containers/deploying-sandboxed-container-workloads.adoc:44]:
```suggestion
* When using a disconnected cluster on a restricted network, you must xref:../operators/admin/olm-configuring-proxy-support.adoc#olm-configuring-proxy-support[configure proxy support in Operator Lifecycle Manager] to access the OperatorHub. Using a proxy allows the cluster to fetch the {sandboxed-containers-operator}.
```

**Review by sagidlow — COMMENTED** [review]
One little nit comment. In the future, make sure to specify which releases this PR is for in the original comment. Otherwise, looks great!

File: sandboxed_containers/deploying-sandboxed-container-workloads.adoc [hunk 2]
```diff
@@ -37,3 +37,8 @@ include::modules/sandboxed-containers-triggering-installation-kata-runtime.adoc[
 include::modules/sandboxed-containers-selecting-nodes.adoc[leveloffset=+3]
 include::modules/sandboxed-containers-scheduling-workloads.adoc[leveloffset=+3]
 include::modules/sandboxed-containers-viewing-workloads-from-cli.adoc[leveloffset=+2]
+
+== Additional resources
```
Comment (zanetworker) [inline sandboxed_containers/deploying-sandboxed-container-workloads.adoc:41]: I think this should be mentioned earlier. Maybe under "Preparing your cluster for OpenShift sandboxed container". Additional resources here to me would mean something that is not related but helpful.

Comment (jboxman) [inline sandboxed_containers/deploying-sandboxed-container-workloads.adoc:41]: For assemblies, we're including an id here:

`[id="{context}_additional-resources"]`

File: sandboxed_containers/deploying-sandboxed-container-workloads.adoc (hunk 1 above)
Comment (MirzWeiss) [reply sandboxed_containers/deploying-sandboxed-container-workloads.adoc:44]: Hey @sagidlow - thanks! I copied this content from another location in the help, but I'll make sure to remove it. :) Once I do, do I submit the PR for peer review again?

Comment (MirzWeiss) [reply sandboxed_containers/deploying-sandboxed-container-workloads.adoc]: Updated.

File: sandboxed_containers/deploying-sandboxed-container-workloads.adoc (hunk 2 above)
Comment (MirzWeiss) [reply sandboxed_containers/deploying-sandboxed-container-workloads.adoc:42]: Added. Thanks!

**Pull request merged**